naucse/templates/index.html

```diff
@@ -9,28 +9,28 @@
     <div class="card card-inverse space motivation">
       <img class="card-img" src="{{url_for('static', filename='pycon2016cz.png')}}" alt="Card image">
       <div class="card-img-overlay">
-        <p class="responsive-text card-text center">Nauč se Python, staň se superhrdinou nebo superhrdinkou a zachraň svět.<br>
-            Python je moderní programovací jazyk. Je univerzální – pohání weby i rakety.
-            Dobře se čte a dá se velice rychle naučit.</p>
+        <p class="responsive-text card-text center">Nauč se Python, staň se superhrdinou nebo superhrdinkou a&nbsp;zachraň svět.<br>
```

Comment:
It sure is. But I'd recommend to keep visible entities, because in many editors this is no different from normal space and it's confusing as hell for a coder.

Other way to keep cleaner text would be a postprocessing that automatically replaces spaces. Kind of like Texy! does.